Sources/PackageLoading/ToolsVersionLoader.swift

```diff
@@ -36,15 +36,15 @@ public class ToolsVersionLoader: ToolsVersionLoaderProtocol {
         public var description: String {
             switch self {
             case .malformed(let versionSpecifier, let file):
-                let file = file.appending(component: ToolsVersion.toolsVersionFileName)
+                let file = Manifest.versionSpecificManifest(path: file)
```

Comment:
Instead of re-computing here, we can directly pass the file path from `load(at:fileSystem)` method